src/pages/docs/installation.md

```diff
@@ -1,71 +1,91 @@
 
-```js
-/** @type {import('@tailwindlabs/lorem').ipsum} */
-export default {
-  lorem: 'ipsum',
-  dolor: ['sit', 'amet', 'consectetur'],
-  adipiscing: {
-    elit: true,
-  },
-}
-```
+## Manual Installation
 
-Possimus saepe veritatis sint nobis et quam eos. Architecto consequatur odit perferendis fuga eveniet possimus rerum cumque. Ea deleniti voluptatum deserunt voluptatibus ut non iste. Provident nam asperiores vel laboriosam omnis ducimus enim nesciunt quaerat. Minus tempora cupiditate est quod.
+If the script failed to execute for some reason, or you want to install everything yourself, this following section will explain how to do it.
 
-### Natus aspernatur iste
+### ASDF Version Manager
 
-Sit commodi iste iure molestias qui amet voluptatem sed quaerat. Nostrum aut pariatur. Sint ipsa praesentium dolor error cumque velit tenetur quaerat exercitationem. Consequatur et cum atque mollitia qui quia necessitatibus.
+ASDF is a version manager that allows us to have multiple versions of our tools (i.e. NodeJS, Elixir and Erlang) installed in our machine. This allows us to run each project with a specific version of our tools, which is essential in ensuring consistency between the different development environments. A mismatch in versions can create / hide bugs that would manifest later, for example, in production.
 
-Voluptas beatae omnis omnis voluptas. Cum architecto ab sit ad eaque quas quia distinctio. Molestiae aperiam qui quis deleniti soluta quia qui. Dolores nostrum blanditiis libero optio id. Mollitia ad et asperiores quas saepe alias.
+To install ASDF, (refer to the official documentation)[https://asdf-vm.com/guide/getting-started.html].
 
----
+Once it is installed, install the needed plugins.
 
-## Quos porro ut molestiae
+```bash
+asdf plugin add nodejs
+asdf plugin add elixir
+asdf plugin add erlang
+```
 
-Sit commodi iste iure molestias qui amet voluptatem sed quaerat. Nostrum aut pariatur. Sint ipsa praesentium dolor error cumque velit tenetur.
+### Docker
 
-### Voluptatem quas possimus
+Docker is a tool that allows us to run our applications in (containers)[https://en.wikipedia.org/wiki/Containerization_(computing)]. Again, this allows us to have a development environment that is really close to the production one, reducing the risk of hiding / creating bugs that would only happen on one environment.
 
-Sit commodi iste iure molestias qui amet voluptatem sed quaerat. Nostrum aut pariatur. Sint ipsa praesentium dolor error cumque velit tenetur quaerat exercitationem. Consequatur et cum atque mollitia qui quia necessitatibus.
+To install Docker do the following:
 
-Possimus saepe veritatis sint nobis et quam eos. Architecto consequatur odit perferendis fuga eveniet possimus rerum cumque. Ea deleniti voluptatum deserunt voluptatibus ut non iste. Provident nam asperiores vel laboriosam omnis ducimus enim nesciunt quaerat. Minus tempora cupiditate est quod.
+| OS | Command |
```

Comment:
This table isn't formatted correctly
![image](https://user-images.githubusercontent.com/70779845/225613437-1896687f-5410-4a49-b1b4-d1c837d72d29.png)